Storage/Processor/Image.php

```diff
@@ -97,6 +97,21 @@ private function getWidthHeight(array $size): array
         return [$width, $height];
     }
 
+    private function fillBackgroundColor($temp)
+    {
+        $background = $this->config->getBackground();
+        imagesavealpha($temp, true);
+
+        $solid_colour = imagecolorallocatealpha(
```

Comment:
No issues with PHPcs in this _ variable? Should be camelCase

Reply:
Agree, but I didn’t face issue word phpcs. I’ll fit it

Reply:
Solved